src/components/dashboard/dashboard.service.ts

```diff
@@ -62,7 +62,12 @@ export class DashboardService implements OnDestroy {
                 this.setLayoutData(layout);
             });
 
-        this.stacked$.pipe(takeUntil(this._onDestroy), filter(stacked => stacked === true)).subscribe(this.updateWhenStacked.bind(this));
+        this.stacked$
```

Comment:
Just noticed `takeUntil` is not the last operator, see the info on our Tips and Tricks page: https://github.houston.softwaregrp.net/caf/ux-aspects-team/wiki/Tips-and-Tricks#operators

I notice the subscriptions on the line below also do not have `takeUntil` as the last operator, this should also be changed